templates/workflow/workflow.tpl

```diff
@@ -50,12 +50,21 @@
 				</span>
 			</h1>
 			<template slot="actions">
-				<pkp-button
-					v-if="submission.status === getConstant('STATUS_PUBLISHED')"
+				<pkp-button v-if="submission.status === getConstant('STATUS_PUBLISHED')"
 					element="a"
 					:label="i18n.view"
 					:href="submission.urlPublished"
 				></pkp-button>
+				<pkp-button v-else-if="submission.status !== getConstant('STATUS_PUBLISHED') && submission.stageId >= getConstant('WORKFLOW_STAGE_ID_EDITING')"
+					element="a"
+					:label="i18n.preview"
+					:href="submission.urlPublished"
+				></pkp-button>
+				<pkp-button v-if="submission.status === getConstant('STATUS_PUBLISHED') && workingPublication.status !== getConstant('STATUS_PUBLISHED')"
+					element="a"
+					:label="i18n.preview"
+					:href="workingPublication.urlPublished"
+				></pkp-button>
```

Comment:
I think for this preview to make sense for a separate version, we need to put the buttons down in the publication tab, next to where we Publish / Schedule for Publication. Can you make that change here and in the `master` PR?

Reply:
Done!